pkg/backend/openshiftcluster/create.go

```diff
@@ -325,8 +325,8 @@ func getRHCOSImage(ctx context.Context) (*azuretypes.Image, error) {
 	}, nil
 }
 
-func randomLowerCaseAlphanumericString(n int) (string, error) {
-	return randomString("abcdefghijklmnopqrstuvwxyz0123456789", n)
+func randomLowerCaseAlphanumericStringWithNoVowels(n int) (string, error) {
```

Comment:
> WithNoVowels

🧐

Reply:
you can't have a naughty word without vowels...